### PR TITLE
Responsive Filter on Products page

### PR DESCRIPTION
**Pull request by himadri2110**
- Made the filters responsive for mobile devices 
- Added the functionality to toggle filter using JS
- Modified styling of cards on Products, Wishlist, and Cart page.

### CODE REVIEW

File: css/cart.css [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 :root {
   --bg-grey: rgb(248, 248, 248);
+  --box-shadow: 2px 2px 5px 2px var(--bg-color);
```
Comment (pritam-kr) [inline css/cart.css:8]: you can use rem too.

**Review by pritam-kr — APPROVED** [review]
All looks good you can merge it.

File: css/cart.css (hunk 1 above)
Comment (himadri2110) [reply css/cart.css:8]: Okay, thank you!

**Review by adidoshi — APPROVED** [review]
This looks great, filter & dismiss icon idea is good.👍 Just a small suggestion if we can add comments to js file as well, as we were told.

**Comment by himadri2110** [comment]
> This looks great, filter & dismiss icon idea is good.👍 Just a small suggestion if we can add comments to js file as well, as we were told.

Yes, thanks!

**Pull request merged**